src/themes/default/pages/Static.vue

```diff
@@ -59,19 +57,24 @@ export default {
       this.activeComponent = component
```

Comment:
we need to remove also this method and `@click.native="setContent(page.component)"` from router-link in Static.vue